api/state.go

```diff
@@ -150,28 +150,7 @@ func (st *state) setLoginResult(p *LoginResultParams) error {
 	st.controllerTag = ctag
 	st.controllerAccess = p.controllerAccess
 	st.modelAccess = p.modelAccess
-
-	hostPorts := p.servers
-	// if the connection is not proxied then we will add the connection address
-	// to host ports
-	if !st.IsProxied() {
-		hostPorts, err = addAddress(p.servers, st.addr)
-		if err != nil {
-			if clerr := st.Close(); clerr != nil {
-				err = errors.Annotatef(err, "error closing state: %v", clerr)
-			}
-			return err
-		}
-	}
-	st.hostPorts = hostPorts
-
-	if err != nil {
-		if clerr := st.Close(); clerr != nil {
-			err = errors.Annotatef(err, "error closing state: %v", clerr)
-		}
-		return err
-	}
-	st.hostPorts = hostPorts
```

Comment:
This removal is reflected by the changed docstring in `api/interface.go` on the HostPort method (see the comment there to better understand this).